specifyweb/frontend/js_src/lib/components/WorkBench/hooks.ts

```diff
@@ -30,56 +33,66 @@ export function getHotHooks(wbView: WbView) {
      * cases
      *
      */
-    afterRenderer: (td, visualRow, visualCol, property, _value) => {
-      if (wbView.hot === undefined) {
+    afterRenderer: (
+      td: any,
+      visualRow: number,
+      visualCol: number,
+      property: number | string,
+      _value: string
```

Comment:
but also, you shouldn't need to provide these types at all here
since TypeScript knows that this entire object must be of type `Partial<Events>` , then it can infer the function argument types from that - no need for you to do that manually

```suggestion
      td,
      visualRow,
      visualCol,
      property,
      _value
```